### ISSUE TITLE
Add a Medium-style static footer that reminds people to sign up

### ISSUE THREAD

**Issue by tanius** [edgeryders/discourse#159]
It was proposed by @ladyniasan that the site would profit from nudging people more to create an account and participate in the discussion.

This should take the form of a `static`-positioned banner-like element at the bottom of the screen, with a call to action and on its right a button to sign up. The button to sign up can be just the same pseudo-button we use for our top-right signup button, pointing to [communities.edgeryders.eu/signup](https://communities.edgeryders.eu/signup).

For a template how this could look, and the inspiration where this idea comes from, see [how it looks on medium.com](https://medium.com/intuitionmachine/dc303056171) without being logged in.

Since this banner is really annoying when reading content meant mostly for anonymous users (like [Autarky Library](https://edgeryders.eu/t/8791)), we need a solution to hide the banner in these cases. Either provide a close button on the banner to hide it until visiting another URL on edgeryders.eu (probably that's the simplest option). Or show the banner only in the [Wellbeing in Europe](https://edgeryders.eu/c/wellbeing) and [Internet of Humans](https://edgeryders.eu/c/ioh) categories.

**Comment by damingo** [comment]
10fe0b6ff00dac3bb0a821f65c59a4a694077756

**Comment by tanius** [comment]
An update about the current implementation of this feature.

The starting point is this: there is a signup call-to-action box that is part of the standard Discourse installation and [it can be enabled here](https://edgeryders.eu/admin/site_settings/category/all_results?filter=signup%20cta). It is however not very attention-seeking because (1) it is only displayed to _returning_ anonymous users, (2) it is shown with an unobtrusive blue background and (3) it is only shown at the end of a topic and not with static positioning at the bottom of the viewport.

* We fixed (1) so that it is now displayed to all anonymous users, both first-time and returning ones. Test it by opening any topic on [edgeryders.eu](https://edgeryders.eu/) in a new incognito browser tab.

* We fixed (2) by changing the background color to "Discourse yellow".

* However, (3) has no proper solution because the space at the bottom of the viewport is already taken by the cookie consent banner for anonymous users. Placing it vertically higher than that banner does not work either as the space taken by the cookie banner changes depending on screen resolution.

So we propose to leave things like this right now. As a last resort option, if desired, we could change the cookie banner plugin to include both the messages about cookies and about signup, and two buttons. The cookie-related button would disappear when clicking "Accept", the other only when being logged in. That's for @ladyniasan to decide.

In the current implementation, the texts of the signup call-to-action box can be adapted by any admin user ([here](https://edgeryders.eu/admin/customize/site_texts/js.signup_cta.intro) and [here](https://edgeryders.eu/admin/customize/site_texts/js.signup_cta.value_prop)).
